### PR TITLE
Merge Reactions With Same Stoichiometry But Different Genes

### PR DESCRIPTION
**Pull request by Devlin-Moyer**
#### Main improvements in this PR:
Implements changes initially proposed in #580 and recently mentioned again in #630:
- Removes `MAR02358` for being a duplicate of `MAR08482`
- Makes `MAR08482` reversible
- Removes `MAR20066` for being a duplciate of `MAR03984`
- Removes `MAR20020` for being a duplicate of `MAR04521`
- Removes `MAR20002` for being a duplicate of `MAR06631`
- Removes `MAR00030` for being a duplicate of `MAR00025`
- Merges annotations, references, E.C. codes, etc. from removed reactions into kept reactions

**I hereby confirm that I have:**
- [X] Tested my code on my own computer for running the model
- [X] Selected `develop` as a target branch
- [X] Any removed reactions and metabolites have been moved to the corresponding deprecated identifier lists

### CODE REVIEW

**Comment by Devlin-Moyer** [comment]
I'm reasonably confident that the YAML validation failed because `MAR00030` was the only reaction that `ENSG00000072310`, `ENSG00000100979`, and `ENSG00000138823` were associated with; should I just go ahead and remove those genes as well? None of them appear to be enzymes, since `MAR00030` wasn't really a "real" metabolic reaction.

**Comment by mihai-sysbio** [comment]
There's a [pending PR](https://github.com/SysBioChalmers/Human-GEM/pull/635) that could be merged to `develop` which would add printing of the errors, thus increasing the certainty. Not sure how urgent this PR is - waiting a bit might help.

**Comment by Devlin-Moyer** [comment]
It's not urgent, and while that would be nice, using this

```python
import cobra
model = cobra.io.load_yaml_model('model/Human-GEM.yml')
[m.id for m in model.metabolites if len(m.reactions) == 0]
[g.id for g in model.genes if len(g.reactions) == 0]
```

was pretty easy; the first list was empty and the second one was exactly the list of genes I mentioned above

File: model/Human-GEM.yml [hunk 1]
```diff
@@ -68370,7 +68370,7 @@
       - gene_reaction_rule: "ENSG00000131482 or ENSG00000141349 or ENSG00000152254"
       - rxnNotes: ""
       - rxnFrom: "HMRdatabase"
-      - eccodes: "3.1.3.9"
+      - eccodes: "3.1.3.9; 3.1.3.58"
```
Comment (edkerk) [inline model/Human-GEM.yml:68373]:
```suggestion
      - eccodes: "3.1.3.9"
```
[EC3.1.3.9](https://www.metacyc.org/META/NEW-IMAGE?type=EC-NUMBER&object=EC-3.1.3.9) is the canonical glucose 6-phosphatase activity, that this reactions also represents. [EC3.1.3.58](https://www.brenda-enzymes.org/enzyme.php?ecno=3.1.3.58) is a less specific sugar-phosphohydrolase found in some lactic acid bacteria.

File: model/Human-GEM.yml [hunk 2]
```diff
@@ -162317,10 +162301,10 @@
       - gene_reaction_rule: "ENSG00000121691 or ENSG00000166016"
       - rxnNotes: ""
       - rxnFrom: "HMRdatabase"
-      - eccodes: "1.11.1.6"
+      - eccodes: "1.11.1.6;1.11.1.21"
```
Comment (edkerk) [inline model/Human-GEM.yml:162304]:
```suggestion
      - eccodes: "1.11.1.6"
```
[EC1.11.1.6](https://www.metacyc.org/META/NEW-IMAGE?type=EC-NUMBER&object=EC-1.11.1.6) is the canonical catalase activity that is also meant here. [EC1.11.1.21](https://www.metacyc.org/META/NEW-IMAGE?type=EC-NUMBER&object=EC-1.11.1.21) a less-specific donor:hydrogen-peroxide oxidoreductase, which can use H2O2 as donor., found in [bacteria and fungi](https://www.brenda-enzymes.org/enzyme.php?ecno=1.11.1.21).

Comment (Devlin-Moyer) [reply model/Human-GEM.yml:162304]: Oh whoops I missed what the difference between those buttons was; I'll take care of that

**Review by haowang-bioinfo — APPROVED** [review]

**Pull request merged**